client/src/AppBundle/Resources/assets/javascripts/pages/overview.js

```diff
@@ -0,0 +1,6 @@
+import ButtonToggler from '../modules/buttonToggler'
+
+window.addEventListener('DOMContentLoaded', () => {
+  const bt = new ButtonToggler()
+  bt.addToggleButtonEventListener('confirmReview', 'edit-report-button')
```

Comment:
I'd prefer to follow the [Design System pattern](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/all.js) here, for a bit of consistency.
```suggestion
  document.querySelector('[data-module="opg-button-toggler"]').forEach($toggler => {
    new ButtonToggler($toggler).init();
  });
```

Though this would require changes to the toggle JavaScript and the HTML markup.